.github/workflows/package.yml

```diff
@@ -67,8 +67,26 @@ jobs:
             autoconf-archive \
             devscripts \
             python3-pip \
-            qemu-user-static \
             ruby
+      - name: Install Qemu from Groovy
+        run: |
+          sudo tee /etc/apt/preferences.d/qemu <<EOF
+          Package: *
+          Pin: release n=focal
+          Pin-Priority: 900
+          Package: *
+          Pin: release n=groovy
+          Pin-Priority: 400
+          EOF
+          cat /etc/apt/preferences.d/qemu
```

Comment:
Why is `cat` needed? `tee` also shows content to stdout.

Reply:
Ah... `cat` is needless.
I'll fix it.